components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/factory/RegistrationRequestFactory.java

```diff
@@ -60,6 +57,8 @@
 public class RegistrationRequestFactory extends HttpIdentityRequestFactory {
 
     private static Log log = LogFactory.getLog(RegistrationRequestFactory.class);
+    public static final String IMPLICIT = "implicit";
```

Comment:
Shall we move these to a place where other constants also resides. These will be required for repetitive use.